meta-freescale-mbl/recipes-bsp/u-boot/u-boot-fslc_%.bbappend

```diff
@@ -1,15 +1,19 @@
-# Copyright (c) 2018 Arm Limited and Contributors. All rights reserved.
+# Copyright (c) 2018-2019 Arm Limited and Contributors. All rights reserved.
 #
 # SPDX-License-Identifier: MIT
 
 # MBL_UBOOT_VERSION should be updated to match version pointed to by SRCREV
 MBL_UBOOT_VERSION = "2018.11-rc1"
 
 SRCREV = "c0c4ee5fce01ec0818c4f27ce029d9b16c8849ad"
+SRCREV_imx7d-pico-mbl = "c72a23701bd9b15dab35e0d43c468ab051739af3"
```

Comment:
Why warp7 and pico7 can't have the same SHA?

Reply:
There are several warp7 specific patches in u-boot, which are to be pushed to upstream.